Fix a mis-placed #endif (in emutls.c from recent libgcc fix) that is breaking the arm-none-eabi (and chromiumos-sdk) builds.

This is temporary, until the GCC repo itself can be fixed.

Author: cmtice@google.com
Date: Feb. 3, 2018
diff --git a/gcc-4.9/libgcc/emutls.c b/gcc-4.9/libgcc/emutls.c
index 77550e057..cba9c3b03 100644
--- a/gcc-4.9/libgcc/emutls.c
+++ b/gcc-4.9/libgcc/emutls.c
@@ -89,7 +89,6 @@ emutls_init (void)
     abort ();
   emutls_key_created = 1;
 }
-#endif
 
 __attribute__((destructor))
 static void
@@ -98,6 +97,7 @@ unregister_emutls_key (void)
   if (emutls_key_created)
     __gthread_key_delete (emutls_key);
 }
+#endif
 
 static void *
 emutls_alloc (struct __emutls_object *obj)
